### PR TITLE
apps/bttester: add support for testing Periodic Advertising

### PR DESCRIPTION
**Pull request by KKopyscinski**

### CODE REVIEW

File: apps/bttester/src/gap.c [hunk 1]
```diff
@@ -1643,6 +1733,150 @@ static void set_filter_accept_list(const uint8_t *data, uint16_t len)
...
+								   gap_event_cb, NULL);
+	if (rc) {
+		SYS_LOG_ERR("Failed to configure extended advertiser; rc=%d", rc);
+		status = BTP_STATUS_FAILED;
+		goto rsp;
+	}
+
+	rc = ble_gap_periodic_adv_configure(1, &params);
+    if (rc) {
+		SYS_LOG_ERR("Failed to configure periodic advertiser; rc=%d\n params.itvl_min %d\n params.itvl_max %d\n", rc, params.itvl_min, params.itvl_max);
```
Comment (michal-narajowski) [inline apps/bttester/src/gap.c:1768]: Fix formatting and too long line.

File: apps/bttester/src/gap.c [hunk 2]
```diff
@@ -1643,6 +1733,150 @@ static void set_filter_accept_list(const uint8_t *data, uint16_t len)
...
+
+	rc = ble_gap_ext_adv_start(1, 0, 0);
+	if (rc) {
+		SYS_LOG_ERR("Failed to start extended advertiser; rc=%d", rc);
+		status = BTP_STATUS_FAILED;
+	}
+
+	rc = ble_gap_periodic_adv_start(1);
+    if (rc) {
+		SYS_LOG_ERR("Failed to start periodic advertiser; rc=%d", rc);
```
Comment (michal-narajowski) [inline apps/bttester/src/gap.c:1791]: Indentation.

File: apps/bttester/src/gap.c [hunk 3]
```diff
@@ -1643,6 +1733,150 @@ static void set_filter_accept_list(const uint8_t *data, uint16_t len)
...
+    uint8_t status = BTP_STATUS_SUCCESS;
+    struct gap_periodic_adv_create_sync_cmd *cmd = (void *) data;
+    struct ble_gap_periodic_sync_params params;
+	struct ble_gap_disc_params scan_params = {0};
+	int rc;
+
+    params.reports_disabled = cmd->reports_disabled;
+    params.skip = cmd->skip;
+    params.sync_timeout = cmd->sync_timeout;
+	SYS_LOG_DBG("\nreports_disabled %d\n skip %d \n sync_timeout %d\n", params.reports_disabled, params.skip, params.sync_timeout);
```
Comment (michal-narajowski) [inline apps/bttester/src/gap.c:1839]: Indentation.

**Review by michal-narajowski — APPROVED** [review]
Minor comments, otherwise LGTM.

**Comment by KKopyscinski** [comment]
I created an issue to track problem with GAP/PADV/PASE/BV-01-C: https://github.com/apache/mynewt-nimble/issues/1512

**Review by sjanc — APPROVED** [review]

**Pull request merged**